src/entry_point.py

```diff
@@ -110,7 +116,7 @@ def __call__(self, parser, namespace, values, option_string=None):
         f"Defaults to {DEFAULT_NUM_PROCESSORS}.",
     )
 
-    g = p.add_mutually_exclusive_group(required=True)
+    g = p.add_mutually_exclusive_group()
```

Comment:
We'd need to add checks to make this required manually if `uninstall` is not in the ARGV.

Reply:
I added that into the same function and created tests to check.